docs/source/install/deb.rst

```diff
@@ -25,8 +32,11 @@ Setup repositories
 
   .. code-block:: bash
 
+    # Pick OS version from: trusty, jessie, wheezy
+    export DISTRO=trusty
     wget -qO - https://bintray.com/user/downloadSubjectPublicKey?username=bintray | sudo apt-key add -
-    echo "deb https://dl.bintray.com/stackstorm/`lsb_release -cs`_staging stable main" | sudo tee /etc/apt/sources.list.d/st2-stable.list
```

Comment:
do you mean `"deb https://dl.bintray.com/stackstorm/trusty_staging stable main"`? How do you mean *hardcode*? I am ok with this as we have only tested trusty, but on the other hand there are repos for others, if we don't need them / can't test them, stop generating them? 

@armab @DoriftoShoes hold my hand here tell me exactly how you want it :)